src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -67,10 +61,16 @@
 
     private static final String SYSTEM_DEFAULT_CHARSET = "SYSTEM_DEFAULT";
 
-    /** Number of times we will show launch diagnostics in a newly encountered workspace before going mute to save resources. */
+    /**
+     * Whether to stream stdio from the wrapper script, which should normally not print any.
```

Comment:
Typo or maybe truncated?
```suggestion
     * Whether to stream stdio from the wrapper script, which should normally not print anything.
```

Reply:
Not a typo.

> …normally not print any [stdio]

Reply:
Yeah, that doesn't pass a grammaticality judgement for me, but I'll take your word that it does for you. I'd expect `print to stdio` (or just stdout) or `print any output`.